src/System Application/App/AI/src/Copilot/CopilotAICapabilities.Page.al

```diff
@@ -59,65 +45,156 @@ page 7775 "Copilot AI Capabilities"
                 }
-
-                field(AOAIServiceLocated2; AOAIServiceLocatedLbl)
+                group(AllowedDataMovementOnInfo)
+                {
+                    ShowCaption = false;
+                    Visible = WithinEUDB and AllowDataMovement;
+                    InstructionalText = 'Copilot and agents use the Azure OpenAI Service available within the EU Data Boundary. To keep using these capabilities, you must allow data movement within this boundary.';
+                }
+                group(EUDBAreaDataMovementGroup)
                 {
                     ShowCaption = false;
+                    label(EUDBAreaCaption)
+                    {
+                        ApplicationArea = All;
+                        Caption = 'By allowing data movement, you agree to data being processed by the Azure OpenAI Service within the EU Data Boundary.';
+                    }
+                    field(EUDBAreaDataMovement; AllowDataMovement)
+                    {
+                        ApplicationArea = All;
+                        Caption = 'Allow data movement';
+                        ToolTip = 'Specifies whether data movement across regions is allowed. This is required to enable Copilot in your environment.';
+                        Editable = WithinAOAINotSupported and AllowDataMovementEditable;
+
+                        trigger OnValidate()
+                        begin
+                            UpdateAllowDataMovement();
+                        end;
+                    }
+                    field(EUDBAreaAOAIServiceLocated; AOAIServiceLocatedLbl)
+                    {
+                        ShowCaption = false;
+
+                        trigger OnDrillDown()
+                        begin
+                            Hyperlink('https://go.microsoft.com/fwlink/?linkid=2250267');
+                        end;
+                    }
+                    field(EUDBAreaDataSecurityAndPrivacy; FAQForDataSecurityAndPrivacyLbl)
+                    {
+                        ShowCaption = false;
+
+                        trigger OnDrillDown()
+                        begin
+                            Hyperlink('https://go.microsoft.com/fwlink/?linkid=2298505');
+                        end;
+                    }
+                    field(EUDBAreaDataProcess; DataProcessByAOAILbl)
+                    {
+                        ShowCaption = false;
+
+                        trigger OnDrillDown()
+                        begin
+                            Hyperlink('https://go.microsoft.com/fwlink/?linkid=2298232');
+                        end;
+                    }
+                }
+            }
 
-                    trigger OnDrillDown()
-                    begin
-                        Hyperlink('https://go.microsoft.com/fwlink/?linkid=2250267');
-                    end;
+            group(AOAINotSupportedArea)
```

Comment:
not _completely_ true
consider `AOAIOutOfRegionArea` or similar (and same for the bool variable)